engine-spring/core/src/test/resources/activiti-context.xml

```diff
@@ -0,0 +1,25 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <bean id="processEngineConfiguration"
+          class="org.camunda.bpm.engine.impl.cfg.StandaloneInMemProcessEngineConfiguration"
+          init-method="init">
+
+        <property name="jdbcUrl" value="jdbc:h2:mem:camunda;DB_CLOSE_DELAY=-1" />
+        <property name="jdbcDriver" value="org.h2.Driver" />
+        <property name="jdbcUsername" value="sa" />
+        <property name="jdbcPassword" value="" />
+
+        <property name="processEngineName" value="activitiContextProcessName"/>
+
+        <property name="databaseSchemaUpdate" value="true"/>
+
+        <property name="jobExecutorActivate" value="false"/>
+    </bean>
+
+    <bean id="processEngine" class="org.camunda.bpm.engine.impl.ProcessEngineImpl">
+        <constructor-arg ref="processEngineConfiguration"/>
+    </bean>
+
+</beans>
```

Comment:
Let's format this according to our style guide and add all properties we currently consider best practice for test setups.
```suggestion
<?xml version="1.0" encoding="UTF-8"?>

<beans xmlns="http://www.springframework.org/schema/beans"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">

  <bean id="processEngineConfiguration" class="org.camunda.bpm.engine.impl.cfg.StandaloneInMemProcessEngineConfiguration"
        init-method="init">

    <property name="jdbcUrl" value="jdbc:h2:mem:camunda;DB_CLOSE_DELAY=-1" />
    <property name="jdbcDriver" value="org.h2.Driver" />
    <property name="jdbcUsername" value="sa" />
    <property name="jdbcPassword" value="" />
    
    <!-- Database configurations -->
    <property name="databaseSchemaUpdate" value="true" />
    
    <!-- job executor configurations -->
    <property name="jobExecutorActivate" value="false" />
    
    <!-- turn off metrics reporter -->
    <property name="dbMetricsReporterActivate" value="false" />
    <property name="telemetryReporterActivate" value="false" />
    <property name="taskMetricsEnabled" value="false" />
    
    <!-- provide a differentiating engine name -->
    <property name="processEngineName" value="activitiContextProcessName"/>
  </bean>

  <bean id="processEngine" class="org.camunda.bpm.engine.impl.ProcessEngineImpl">
    <constructor-arg ref="processEngineConfiguration"/>
  </bean>

</beans>
```